src/components/MovieCard.tsx

```diff
@@ -0,0 +1,80 @@
+import {
+  Box,
+  CircularProgress,
+  CircularProgressLabel,
+  Grid,
+  Heading,
+  Image,
+  Text,
+  VStack,
+  GridItem,
+  Stack,
+  Flex,
+} from "@chakra-ui/react";
+
+type MovieCardProps =
+  | { isLoading: true }
+  | {
+      poster_path: string;
+      title: string;
+      release_date: string;
+      vote_count: number;
+      isLoading?: false;
```

Comment:
props `isLoading` ini minta tolong digeser ke paling atas ya, biar enak bedainnya dan lebih rapi


```suggestion
  | {
      isLoading?: false;
      poster_path: string;
      title: string;
      release_date: string;
      vote_count: number;
```

Reply:
Siapp